### PR TITLE
Update from packages.config to PackageReference

### LINKED ISSUE #285: Describe development environment requirements
There should be a part of the readme.md which describes the development environment requirements. It is probably enough just to state Visual Studio version.

### PR DESCRIPTION
**Pull request by onovotny**
Fixes #285

Simplifies the csproj files and enables transitive nuget dependencies

### CODE REVIEW

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.2.0-PullRequest0283.79.build.500 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/21119886) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/b40f915c5a by @onovotny)

**Comment by ErikMogensen** [comment]
This is great! However it will set a requirement of using VS 2017 15.7 or higher for building. This should not be a problem, but we should specify that in the project description, therefore I created #285 for this. When that issue is fixed I'll approve this one.

There are some limitations described at https://docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference, but I do not think we'll be affected.

File: src/ServiceBusExplorer.Tests/ServiceBusExplorer.Tests.csproj [hunk 1]
```diff
@@ -79,16 +58,15 @@
   <ItemGroup>
     <Folder Include="Artifacts\" />
   </ItemGroup>
+  <ItemGroup>
```
Comment (SeanFeldman) [inline src/ServiceBusExplorer.Tests/ServiceBusExplorer.Tests.csproj]: @onovotny did you use a tool? Usually I like to see more concise entries, such as `<PackageReference Include="id" Version="version" />

Comment (onovotny) [reply src/ServiceBusExplorer.Tests/ServiceBusExplorer.Tests.csproj]: I'll fix it up. NuGet has a "migrate packages.config to PackageRef" menu option that I used.

**Comment by onovotny** [comment]
All fixed up and addresses #285 as well.

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.2.0-PullRequest0283.81.build.501 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/21147372) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/74c86b29cf by @onovotny)

**Pull request merged**